src/main/java/com/google/sps/data/GameStage.java

```diff
@@ -18,6 +18,8 @@
    * Game Stages are levels on the gameboard. Each level can be accessed by a player, but only one
    * at a time.
    */
+  private static final String DEFAULT_NEXTID_IF_FINAL_STAGE = "nonextid";
```

Comment:
Its a very funky string, if you're able to hard code it and change it, maybe camel case or with dashes?